apps/sms/locales/sms.en-US.properties

```diff
@@ -135,8 +135,18 @@ discard-new-message         = Discard the message you’re composing and open th
 resend-confirmation         = The message could not be sent. Try again?
 resend-message              = Resend this message
 deleteMessage-confirmation  = Delete selected message?
-deleteMessages-confirmation = Delete selected messages?
-deleteThreads-confirmation2 = Delete selected message threads?
+deleteMessages-confirmation-message = {[ plural(n) ]}
+deleteMessages-confirmation-message[one] = Delete selected message?
+deleteMessages-confirmation-message[two] = Delete {{n}} selected messages?
+deleteMessages-confirmation-message[few] = Delete {{n}} selected messages?
+deleteMessages-confirmation-message[many] = Delete {{n}} selected messages?
+deleteMessages-confirmation-message[other] = Delete {{n}} selected messages?
+deleteThreads-confirmation-message = {[ plural(n) ]}
+deleteThreads-confirmation-message[one] = "Delete selected message thread?"
```

Comment:
Sorry, My mistake, I forgot to remove them after I copied and pasted there.
Thanks :)